### PR TITLE
Correct some instructions in README.namelist

### PR DESCRIPTION
**Pull request by smileMchen**
TYPE: text only

KEYWORDS:  README.namelist, NMM

SOURCE: internal

DESCRIPTION OF CHANGES: 
A few physics options are declared as "NMM only" in README.namelist, whereas they actually work 
for both ARW and NMM. 
Surface Layer: NCEP GFS (`sf_sfclay_physics=3`)
PBL: Hybrid EDMF GFS (`bl_pbl_physics=3`)
This PR corrects the information. 

LIST OF MODIFIED FILES: 
M    run/README.namelist

TESTS CONDUCTED: 
A single ARW case is successfully done with the options of sf_sfclay_physics=3 and bl_pbl_physics=3. Results look fine.

### CODE REVIEW

File: run/README.namelist [hunk 1]
```diff
@@ -656,7 +656,7 @@ Namelist variables for controlling the adaptive time step option:
                                      = 0, no surface-layer
                                      = 1, Revised MM5 Monin-Obukhov scheme (Jimenez, renamed in v3.6)
                                      = 2, Monin-Obukhov (Janjic) scheme
-                                     = 3, NCEP Global Forecast System scheme (NMM only)
+                                     = 3, NCEP Global Forecast System scheme (works for both ARW and NMM )
```
Comment (davegill) [inline run/README.namelist]: Ming,
We only seem to comment when the scheme works for only a single dynamical core. In this mod and the next, remove the parenthetical comment.

**Comment by davegill** [comment]
@smileMchen 
Ming,
Are there any problems in check_a_mundo that need to be addressed for this? If not, please add that info to the commit message, so everyone knows that issue was considered.

**Comment by barlage** [comment]
@davegill Maybe this doesn't fit in this PR, but it seems to apply to the title anyway... after we added the functions to the new urban memory (PR #878 ), is the section of README.namelist that includes the urban dimensions (num_urban_ndm, etc.) still necessary?

**Comment by weiwangncar** [comment]
I agree they should be removed. We should open another PR for this.

**Comment by davegill** [comment]
@smileMchen 
Ming
>  Maybe this doesn't fit in this PR, but it seems to apply to the title anyway... after we added the functions to the new urban memory (PR #878 ), is the section of README.namelist that includes the urban dimensions (num_urban_ndm, etc.) still necessary?

 - [ ] Ming would you take a look at this query?

**Pull request merged**